### PR TITLE
chore: example run script updates

### PR DESCRIPTION
**Pull request by vaind**
## :scroll: Description

Some updates to the flutter example when working on source-context support

#skip-changelog

## :pencil: Checklist

- [x] I reviewed submitted code
- [ ] I added tests to verify changes
- [ ] I updated the docs if needed
- [x] All tests passing
- [x] No breaking changes

### CODE REVIEW

File: flutter/example/.gitignore [hunk 1]
```diff
@@ -31,9 +31,6 @@
 .pub/
 /build/
 
-# Web related
-lib/generated_plugin_registrant.dart
```
Comment (vaind) [inline flutter/example/.gitignore:35]: flutter removed this 🤷

File: flutter/example/pubspec.yaml [hunk 2]
```diff
@@ -33,9 +33,10 @@ flutter:
...
 sentry:
-  upload_native_symbols: false
+  upload_native_symbols: true
   upload_source_maps: true
+  include_native_sources: true
   project: sentry-flutter
   org: sentry-sdks
-  log_level: error
   wait_for_processing: true
+  commits: false
```
Comment (marandaneto) [inline flutter/example/pubspec.yaml]: Any reason why disabling this feature here?

File: flutter/example/web/index.html [hunk 3]
```diff
@@ -31,4 +32,5 @@
   </script>
   <script src="main.dart.js" type="application/javascript"></script>
 </body>
-</html>
+
+</html>
```
Comment (marandaneto) [inline flutter/example/web/index.html]:
```suggestion
</html>

```

**Review by marandaneto — APPROVED** [review]
Thanks @vaind

**Pull request merged**